fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/config/FineractProperties.java

```diff
@@ -72,4 +75,21 @@ public static class FineractCorrelationProperties {
         private boolean enabled;
         private String headerName;
     }
+
+    @Getter
+    @Setter
+    public static class FineractPartitionedJobProperties {
+
+        private List<JobProperty> jobProperties;
+    }
+
+    @Getter
+    @Setter
+    public static class JobProperty {
```

Comment:
It would be more consistent to name this class `PartitionedJobProperty` see also my comments above.

If you take all these changes together then the configuration becomes significantly shorter and less repetitive (in the naming).